llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1129,6 +1137,10 @@ void VPlanTransforms::truncateToMinimalBitwidths(
                VPWidenSelectRecipe, VPWidenLoadRecipe>(&R))
         continue;
 
+      // TODO: Figure out how to handle multiple defs.
+      if (R.getNumDefinedValues() > 1)
+        continue;
```

Comment:
`truncateToMinimalBitwidths()` may also need to be updated to support multiple defs.